.github/workflows/deploy.yml

```diff
@@ -25,6 +25,7 @@ jobs:
       - name: Checkout Package
         uses: actions/checkout@v2
         with:
+          ref: ${{ github.ref_name }}
```

Comment:
Could tell me which branch is this `ref_name` referring to? and where we could access this variable?

Reply:
The `ref_name` will change based on which branch this workflow was triggered on. You can find more information about it under GitHub context here: https://docs.github.com/en/actions/learn-github-actions/contexts#github-context

Documentation:


github.ref_name | string | The branch or tag name that triggered the workflow run.
-- | -- | --

Reply:
Yeah according to the [doc](https://github.com/actions/checkout#usage):
> 
```
- uses: actions/checkout@v2
  with:
    # Repository name with owner. For example, actions/checkout
    # Default: ${{ github.repository }}
    repository: ''

    # The branch, tag or SHA to checkout. When checking out the repository that
    # triggered a workflow, this defaults to the reference or SHA for that event.
    # Otherwise, uses the default branch.
    ref: ''
    ...
```

It is default to the main branch and will change based on which event triggers this workflow. Do we need to declare it explictly?